File: 진우/3weeks/5597. 과제 안 내신 분．．？/과제 안 내신 분．．？.py
num = [0] * 31  # 1~30번을 사용하기 위해 31칸 (0은 안 씀)

for _ in range(28):
    n = int(input())
    num[n] = 1  # 해당 번호를 제출 표시

for i in range(1, 31):
    if num[i] == 0:  # 제출 안 한 학생 출력
        print(i)

File: 진우/3weeks/5597. 과제 안 내신 분．．？/과제 안 내신 분．．？.cc
#include <iostream>
#include <string>
#include <vector>
#include <algorithm>
using namespace std;

int main() {
    vector<int> num(31, 0);
        vector<int> arr(28);
        for (int idx = 0; idx < 28; idx++) {
        cin >> arr[idx];
        num[arr[idx]] = 1;
        }

        for (int i = 1; i < (int)num.size(); i++) {
            if (num[i] == 0) {
                cout << i << endl;
            }
        }

        return 0;
  }

